frontend/app/controllers/login.js

```diff
@@ -0,0 +1,35 @@
+import Ember from "ember"
+import getLoggedInUser from "frontend/mixins/get_logged_in_user";
+import constants from "frontend/utils/constants";
+import ajax from "frontend/helpers/ajax";
+
+let controller = Ember.Controller.extend(getLoggedInUser,{
+  actions:{
+    login:function(){
+      let self = this;
+      this.set('errorMessage', null)
+      let hash= {
+        data: {
+          user:{
+            email: this.get('email'),
+            password: this.get('password')
+          }
+        },
+        success:function(data){
+          if(data.message == "sign_in success"){
```

Comment:
Instead of comparing text, wouldn't it be better to use the request's status code to determine wether login was succesful?